src/main/java/at/hannibal2/skyhanni/features/misc/discordrpc/DiscordRPCManager.kt

```diff
@@ -60,66 +54,55 @@ object DiscordRPCManager : IPCListener {
                 if (isActive()) {
                     return@launch
                 }
-                consoleLog("Starting Discord RPC...")
-                // TODO, change functionality to use enum rather than ordinals
-                firstLine = getStatusByConfigId(config.firstLine.get().ordinal)
-                secondLine = getStatusByConfigId(config.secondLine.get().ordinal)
+                logger.info("Starting Discord RPC...")
                 startTimestamp = System.currentTimeMillis()
-                client = IPCClient(applicationID)
-                client?.setListener(this@DiscordRPCManager)
-
-                try {
-                    client?.connect()
-                    if (fromCommand) ChatUtils.chat(
-                        "Successfully started Rich Presence!",
-                        prefixColor = "§a"
-                    ) // confirm that /shrpcstart worked
-                } catch (ex: Exception) {
-                    consoleLog("Warn: Failed to connect to RPC!")
-                    consoleLog(ex.toString())
-                    ChatUtils.clickableChat(
-                        "Discord Rich Presence was unable to start! " +
-                            "This usually happens when you join SkyBlock when Discord is not started. " +
-                            "Please run /shrpcstart to retry once you have launched Discord.",
-                        onClick = {
-                            startCommand()
-                        }
-                    )
+                client = IPCClient(APPLICATION_ID).apply {
```

Comment:
extract code inside apply into a new function. Too many intendations

Reply:
Ive moved it to its own method and cleaned up some misnaming of a method